### ISSUE TITLE
Item priority

### ISSUE THREAD

**Issue by Schaloth** [stevin05/CraftScan#49]
It would be good if there is a priority checkmark for items like crafters. At the moment I want to track both epic, blue and green items (aka spark, 590 and pvp ones) with their slots like wrist, helm, waist etc. I tried couple keywords to making it give priority to spark ones but no success at all, everytime couple different items not follow the rule. For example if a buyer wrote LF leather(filtered by lw) gloves, I want addon see it as spark one. On some items it shows like that but other ones it detects blue or pvp one.

**Comment by stevin05** [comment]
Yeah, I've had this problem too. If it doesn't find a logical best option, it's just based on lua table order, so basically random. The secondary keywords were meant to help with this, but that still doesn't help if they say something very generic that could match any of the 3 blue/pvp/spark types.

Will see if there's any identifier for spark crafts in item data to make that the default since people usually specify if they mean the pvp or blue ones.

**Comment by Schaloth** [comment]
I noticed that priority for similar items goes to first one in the lua file. For example on my lua file recipes for tailoring start at line 243. There is As a cloth pvp leg at line 244, spark leg at 401 and blue leg at line 448. If I disable pvp leg it detects spark one and if I disable pvp and spark one then it detects blue one as default. As an avarage joe I don't know how the lua file created and wanted to change spark ones location to upper lines in recipe list but everytime it gets re-alligned to default order so no success. Maybe if there is way to change order in recipe list or disabling the re-alligning of recipes it could be solved :D
